fulp_modules/features/sect/bull.dm

```diff
@@ -0,0 +1,34 @@
+/mob/living/basic/cow/bull
+	name = "bull"
+	desc = "Sacred in some communities. Dinner in others."
+	icon = 'fulp_modules/features/sect/earth-cult.dmi'
```

Comment:
files should use underscores instead of dash, less prone to breaking from bad DM stuff